db/migrations/03_create_users_table.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE users
+(
+  id SERIAL PRIMARY KEY,
+  username VARCHAR(60),
+  password VARCHAR(60)
```

Comment:
Could be useful setting a NOT NULL option where is expected to never receive a null value